packages/tasks/src/faucet.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-ignore
```

Comment:
What is this used for?

Reply:
The faucet doesn't ship with types, this tells TS code to not look for types for this package.